code/game/gamemodes/clock_cult/clock_unsorted.dm

```diff
@@ -171,6 +171,10 @@ abnormal spacing, hyphens, and grave accents.
 List of nuances:
 
 - Any time the word "of" occurs, it is linked to the previous word by a hyphen. If it is the first word, nothing is done. (i.e. "V nz-bs Ratvar." directly translates to "I am-of Ratvar.")
+- Any time "th", followed by any two letters occurs, you add an apostrophe between those two letters, i.e; "Thi's"
+- In the same vein, any time "ti", followed by one letter occurs, you add an apostrophe between "i" and the letter, i.e; "Ti'me"
+- Whereever "te" or "et" appear and there is another letter next to the e(i.e; "m"etal, greate"r"), add a hyphen between "e" and the letter, i.e; "M-etal", "Greate-r"
+- if something doesn't look interesting enough, sprinkle hyphens and apostrophes until it looks better. Don't overuse it, though.
```

Comment:
I don't like this rule being loose, you've gone so far as to standardise some grammar rules, but hyphens/apostrophes are "Where ever it looks better"

One thing that was discussed was making ratvarian automated, just for the ~rule of cool~ of it, automation can be done for "after any ti split the next 2 letters" and such like that, but it struggles to do "stick ' and - where it looks nice" (because to automate that bit you'd have to add a -rough- ruling as to where to put it, at which point you may as well just have the rule be official!)